### ISSUE TITLE
Enforce minimum design selection at checkpoint review milestone 

### ISSUE THREAD

**Issue by vikasrohit** [appirio-tech/connect-app#2410]
### Expected behavior
It should not enable the `complete review` button until the required number (right now it is 5) of designs are selected. Only exception is when the available number of options is lower than the required number e.g. if there are only 3 design options to select from, it should enable the `complete review` button only when all 3 are selected.

### Actual behavior
It enables the button even without selecting a single design.

### Steps to reproduce the problem
- Follow the milestones for a design phase and observe the behaviour at checkpoint review milestone.

### Screenshot/screencast
N/A

--

#### Environment
- OS: Any
- Browser (w/version): Any
- User role (client, copilot or manager): Copilot or Manager
- Account used: N/A

**Comment by vic-topcoder** [comment]
I think we should disable "submit for review" for customers as the specification is hidden on the phase cards. @acshields please chime in

**Comment by vic-topcoder** [comment]
Yeah, poor wording. Ignore my previous comment. The button should not be enabled until the customer selects the required number of designs (review, final designs).
